src/main/java/org/candlepin/hostedtest/HostedTestDataStore.java

```diff
@@ -968,7 +968,21 @@ protected SubscriptionsCertificate convertSubscriptionCertificate(CertificateInf
         return null;
     }
 
+    protected void addCloudOfferIdToProductId(String cloudOfferId, String productId) {
```

Comment:
Are offering IDs always 1:1 with product IDs?

Also as a nit: Cloud offering is the owner of this relationship, so the naming should be reversed. We're adding a product to a cloud offering, not the other way around. Additionally, so long as we're maintaining a 1:1 mapping, we're not really "adding" so much as we're setting or mapping.

Reply:
From my understanding the offering IDs are 1:1 with product IDs. I updated the name